syntax = "proto3";



message MessageStatistics
{
  uint32 count = 1;
  uint32 flow_rate_second = 2;
  uint64 flow_rate_total = 3;
  uint32 second = 4;
}
